staff/edu-berenguer/cv/index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>CV</title>
+</head>
+<body>
+    <header>
+        <h1>HOMER SIMPSON</h1>
+        <nav>
+            <ul>
+                <li><a href="#experience">Experiencia</a></li>
+                <li><a href="#studies">Estudios</a></li>
+                <li><a href="#other">Otros</a></li>
+                <li><a href="#lenguages">Idiomas</a></li>
+                <li><a href="#hobbies">Aficiones</a></li>
+            </ul>
+        </nav>
+    </header>
+    <aside id="img">
+        <figure>
+            <img src="./homer.png" alt="foto">
+        </figure>   
+    </aside>
+    <section>
+        <article id="experience">
```

Comment:
may you use ```section``` instead of ```article```, see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article

Reply:
@manuelbarzi  done